modules/home/assets/js/components/home-screen.js

```diff
@@ -1,17 +1,24 @@
 import { Container, Box } from '@elementor/ui';
-
 import TopSection from './top-section';
+import PromotionBar from './side-bar-promotion';
 
 const HomeScreen = ( props ) => {
 	return (
 		// Box wrapper around the Container is needed to neutralize wp-content area left-padding
 		<Box sx={ { pr: 1 } }>
-			<Container disableGutters={ true } maxWidth="lg" sx={ { display: 'flex', flexDirection: 'column', gap: { xs: 1, md: 3 }, py: { xs: 2, md: 6 } } }>
+			<Container disableGutters={ true } maxWidth="lg" sx={ { display: 'flex', flexDirection: 'column', gap: { xs: 1, md: 3 }, pt: { xs: 2, md: 6 }, pb: 2 } }>
 				<TopSection
 					topData={ props.homeScreenData.data.top }
 					createNewPageUrl={ props.homeScreenData.createNewPageUrl }
 				/>
 			</Container>
+			<Container disableGutters={ true } sx={ { display: 'flex', flexDirection: { xs: 'column', sm: 'row' }, justifyContent: 'space-between' } }>
+				{ /* Placeholder container to ensure layout until other components are added*/ }
+				<Container maxWidth="md"></Container>
+				<Container maxWidth="xs" disableGutters={ true } sx={ { width: { md: '305px' } } }>
+					<PromotionBar sideData={ props.homeScreenData.data.sidebar_upgrade } />
+				</Container>
+			</Container>
```

Comment:
Maybe the entire thing should be in the `PromotionBar`? It would read a lot easier:

```
<Box sx={ { pr: 1 } }>
    <TopSection ..>
    <PromotionBar...>
</Box>
```

Reply:
@louiswol94 I think the promotion bar needs to be wrapped inside a container, because there are multiple elements next to each other.

Reply:
Ok, if you say so ;-)